packages/cfd/src/sass/cfd-dashboard.scss

```
@@ -140,97 +119,120 @@
-                &-row {
-                    display: flex;
-                    align-items: baseline;
-                    justify-content: center;
-                    margin-bottom: 3.1rem;
 
-                    > svg:first-child {
-                        margin-bottom: 0.4rem;
-                        margin-right: 0.8rem;
-                    }
-                    a > svg {
-                        margin-top: 0.8rem;
-                        margin-right: 0.8rem;
-                    }
+                @include mobile {
+                    display: block;
                 }
-                &-download {
-                    display: grid;
-                    grid-auto-flow: column;
-                    grid-gap: 0.8rem;
-                    justify-content: center;
+            }
 
-                    a {
-                        margin-top: 0.4rem;
-                    }
+            &--desktop {
+                margin-bottom: 6.4rem;
+            }
+
+            &-icons {
+                text-align: center;
+                margin-bottom: 1.6rem;
+
+                svg {
+                    margin: 0 0.4rem;
+                    display: inline-block;
                 }
             }
-            &--mobile {
-                align-items: center;
+
+            &-download-buttons {
                 display: flex;
-                flex-flow: column nowrap;
-                margin-left: 4.7rem;
+                align-items: center;
+                justify-content: center;
+                flex-wrap: wrap;
 
-                &-devices {
-                    margin-bottom: 1.6em;
-                    svg:first-child {
-                        margin-right: 0.8em;
-                    }
-                }
-                &-link:not(:first-child) {
-                    margin-left: 0.8rem;
-                }
-                @include mobile {
-                    margin-left: unset;
-                    margin-right: unset;
-                    &-links {
-                        display: flex;
-                        flex-direction: column;
-                    }
+                & > a {
+                    display: inline-block;
+                    margin: 0.4rem;
                 }
             }
         }
-        &--heading {
-            @include typeface(--title-center-bold-black, none);
-            color: var(--text-general);
-            line-height: 1.5;
-            margin-left: 2.1rem;
-            margin-bottom: 4.2rem;
-            @include desktop {
-                margin-left: 0;
-                text-align: center;
+
+        &-dxtrade-button {
+            background: $color-black-7;
+            border-radius: $BORDER_RADIUS;
+            text-decoration: none;
+            display: grid;
+            grid-template-columns: 32px 1fr;
+            grid-gap: 0.8rem;
+            width: 12.7rem;
+            align-items: center;
+            height: 4rem;
+            border: 1px solid #a6a6a6;
```

```suggestion
            border: 1px solid #a6a6a6;
```
Color variable if possible

---

Changed.